.changeset/pre.json

```diff
@@ -0,0 +1,19 @@
+{
+  "mode": "pre",
```

Comment:
We're in pre-release mode now! After merging this PR, all new versions will be published as alpha.

Reply:
So excited!